src/components/tx_list.vue

```diff
@@ -8,10 +8,15 @@
 
     <template v-else>
       <q-infinite-scroll ref="scroller" @load="loadMore">
-        <q-list link no-border :dark="theme == 'dark'" class="loki-list tx-list">
+        <q-list
+          link
+          no-border
+          :dark="theme == 'dark'"
+          class="loki-list tx-list"
+        >
           <q-item
-            v-for="tx in tx_list_paged"
-            :key="`${tx.txid}-${tx.type}`"
+            v-for="(tx, i) in tx_list_paged"
```

Comment:
if we have a duplicate `tx.id` then we have an issue 🤔

Reply:
yeah, if the wallet receives a multioutput transaction (and the multiple outputs are the same address) then that wallet will receive multiple inputs, but they'll have the same txid.